knowledge-base/github/super-linter/slim/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Super-Linter slim'
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+  is-default: false
+  permissions:
+    checks: write
+    checks-reason: to update Checks Section # CHECKOUT: https://github.com/github/super-linter#how-to-use
```

Comment:
@h0x0er `super-linter` uses statuses as per my earlier analysis. so `super-linter/slim` might be using the same. You can use the same text as well. 
https://github.com/step-security/secure-workflows/blob/1bd3f566c0c735e3039ae1d58d8763eb6a3f9a7b/knowledge-base/github/super-linter/action-security.yml#L6
https://github.com/github/super-linter/blob/feb45bb0350fec6cc687042ed3385154e72d6f89/lib/linter.sh#L644

Reply:
@varunsh-coder resolved this issue.